src/instrumentsscene/qml/MuseScore/InstrumentsScene/internal/InstrumentsOnScoreView.qml

```diff
@@ -123,7 +123,7 @@ Item {
             navigation.column: 1
 
             icon: IconCode.DELETE_TANK
-            toolTipTitle: qsTrc("instruments", "Remove selected instruments from score")
+            toolTipTitle: root.isInstrumentSelected ? qsTrc("instruments", "Remove selected instruments from score") : qsTrc("instruments", "Remove selected staves from score")
```

Comment:
This is in the Add/Remove instruments dialog, rather than the instruments panel, so there are no staves here, and `root.isInstrumentSelected` doesn't exist in this context.

Reply:
OK, removed again